CMakeLists.txt

```diff
@@ -74,6 +74,9 @@ mark_as_advanced(OPAE_MINIMAL_BUILD)
 option(OPAE_BUILD_TESTS "Enable building of OPAE unit tests" OFF)
 mark_as_advanced(OPAE_BUILD_TESTS)
 
+option(OPAE_WITH_QSFPINFO_QSFPPRINT "Enable qsfpinfo print qsfp" OFF)
```

Comment:
I checkout this branch build built as is, with OPAE_WITH_QSFPINFO_QSFPPRINT OFF, and everything built fine without downloading anything from the ethtool repo.

I performed a clean build with OPAE_WITH_QSFPINFO_QSFPPRINT ON and observed ethtool files being fetched, but I also observed a compilation error:

 61%] Building C object binaries/qsfpinfo/CMakeFiles/qsfpinfo.dir/main.c.o
[ 62%] Building C object binaries/qsfpinfo/CMakeFiles/qsfpinfo.dir/cmis_wrap.c.o
/home/mgerlach/git/opae-sdk/binaries/qsfpinfo/cmis_wrap.c:28:10: fatal error: cmis.c: No such file or directory
   28 | #include "cmis.c"
      |          ^~~~~~~~

Reply:
Hi @fpgamatt Thanks for doing a good testing. A dependency to the target get-ethtool-files that copies the needed ethtool files into the src directory was missing. When I tested, I wrongly just called "make" indstead of "make qsfpinfo"

Reply:
I tested the latest update, and everything builds correctly.